app/src/app/search/page.tsx

```diff
@@ -40,11 +41,15 @@ export default function Search() {
                 return (<tr><td colSpan={4} align='center'>No players found.</td></tr>);
             } else {
                 return results.map((item) => (
-                    <tr style={{cursor:'pointer'}} onClick={event =>  window.location.href=`/playerstats/${item.pid}`} key={item.pid}>
-                        <td align='center'>{item.pid}</td>
-                        <td>{item.firstname}</td>
-                        <td>{item.lastname}</td>
+                    <tr style={{cursor:'pointer'}} key={item.pid}>
+                        <td align='center' onClick={() =>  window.location.href=`/playerstats/${item.pid}`}>{item.pid}</td>
+                        <td onClick={() =>  window.location.href=`/playerstats/${item.pid}`}>{item.firstname}</td>
+                        <td onClick={() =>  window.location.href=`/playerstats/${item.pid}`}>{item.lastname}</td>
+                        <td>
+                            <BookmarksBtn pid={Number(item.pid)} initialValue={bookmarks.includes(Number(item.pid))} />
```

Comment:
bookmarks button probably should not be shown/disabled if the user is not logged in (ie session is null)